src/main/java/org/grouplens/grapht/Instantiators.java

```diff
@@ -87,13 +93,15 @@ public static Instantiator ofProviderInstantiator(Instantiator pinst) {
     public static Provider<?> toProvider(Instantiator instantiator) {
         // First try to unpack the instantiator
         if (instantiator instanceof ProviderInstantiator) {
-            Instantiator itor = ((ProviderInstantiator) instantiator).providerInstantiator;
-            if (itor instanceof InstanceInstantiator) {
-                return (Provider) ((InstanceInstantiator) itor).instance;
-            }
+                Instantiator itor = ((ProviderInstantiator) instantiator).providerInstantiator;
+                if (itor instanceof InstanceInstantiator) {
+                    return (Provider) ((InstanceInstantiator) itor).instance;
+                }
+
```

Comment:
It looks to me like these changes are just whitespace changes, possibly involving tabs and spaces.  Can we revert them so we don't have the diff noise? The original spacing was correct (4-space indents).